module_intros/box.Box.ipynb

```diff
@@ -0,0 +1,656 @@
+{
```

Comment:
<p>Let's not duplicate this. Point directly to <a href="https://freud.readthedocs.io/en/latest/tutorial/periodic.html" target="_blank">https://freud.readthedocs.io/en/latest/tutorial/periodic.html</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>